### ISSUE TITLE
New upstream location

### ISSUE THREAD

**Issue by tskisner** [conda-forge/libsharp-feedstock#3]
Issue:

I heard from the libsharp author (Martin Reinecke) that he has a new release of libsharp at a new location:

https://gitlab.mpcdf.mpg.de/mtr/libsharp

And this version has a change to the API.  Just noting this here.  I am working on getting PR #2 working first, and then will address this new version.

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

**Comment by beckermr** [comment]
The new upstream code is actually a totally different package as far as I can tell. It is also not versioned, and so we can't ship it in conda-forge anyways.

**Comment by tskisner** [comment]
Yes the upstream libsharp2 code is completely different, and the original version used by this feedstock has been archived on github.  I'll reach out to Martin Reinecke (the author) and see if he plans to make a versioned release.

**Comment by beckermr** [comment]
Great! We should make a new feedstock called libsharp2 for this.
